packages/eslint-plugin-pf-codemods/lib/rules/v5/emptyStateIcon-icon-required.js

```diff
@@ -0,0 +1,30 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8737
+module.exports = {
+  meta: {},
+  create: function (context) {
+    const imports = getPackageImports(context, "@patternfly/react-core").filter(
```

Comment:
Since we're only looking for a single component, we could do a `.find` instead of filter here:

```suggestion
    const emptyStateIconImport = getPackageImports(context, "@patternfly/react-core").find(
```